.gitignore

```diff
@@ -7,4 +7,7 @@ dist/*
 .venv
 .env/
 Pipfile
-*.egg-info
\ No newline at end of file
+*.egg-info
```

Comment:
Felipe this code is awesome and elegant

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Unreadable, Not secure, Inefficient, Inelegant